### PR TITLE
1488 download source files

### PR DESCRIPTION
**Pull request by sylmorin-gbif**
Allow source files to be downloaded directly from the manage resource page

### CODE REVIEW

File: src/main/java/org/gbif/ipt/action/portal/ResourceFileAction.java [hunk 1]
```diff
@@ -257,4 +258,33 @@ public String rtf() {
     filename = sb.toString();
     return execute();
   }
+
+  public String rawsource() {
+    if (resource == null) {
```
Comment (mike-podolskiy90) [inline src/main/java/org/gbif/ipt/action/portal/ResourceFileAction.java:263]: I think we can have only one if
`if (resource == null || id == null)`

Comment (sylmorin-gbif) [reply src/main/java/org/gbif/ipt/action/portal/ResourceFileAction.java:263]: Done, but in the other PR (bootstrap): https://github.com/gbif/ipt/pull/1546

**Comment by mike-podolskiy90** [comment]
I close this one. Relevant is https://github.com/gbif/ipt/pull/1546

**Pull request closed without merging**